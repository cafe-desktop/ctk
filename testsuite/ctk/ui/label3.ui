<!-- test invalid label attribute name -->
<interface>
  <object class="CtkLabel">
    <attributes>
      <attribute name="warbl" value="true"/>
    </attributes>
  </object>
</interface>
