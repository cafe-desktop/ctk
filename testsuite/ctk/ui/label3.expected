ERROR: ctk-builder-error-quark 6
.:5:45 Could not parse enum: 'warbl'
